lib/index.js

```diff
@@ -22,13 +20,24 @@ module.exports = {
 
     parser.parse();
 
+    const { root } = parser;
+    const ogToString = root.toString;
+
+    function toString(stringifier) {
+      return ogToString.bind(root)(stringifier || module.exports.stringify);
+    }
+
+    root.toString = toString.bind(root);
+
     return parser.root;
   },
 
-  stringify(node, builder) {
-    const stringifier = new Stringifier(builder);
-    stringifier.stringify(node);
-  },
+  // stringify(node, builder) {
```

Comment:
Should you remove this commented code if it has been replaced?

Reply:
indeed. thanks for pointing that out